sql/021-config-functions.sql

```diff
@@ -301,7 +301,7 @@ $$ LANGUAGE plpgsql;
 
 DROP FUNCTION IF EXISTS cs_add_column_v1(table_name text, column_name text);
 
-CREATE FUNCTION cs_add_column_v1(table_name text, column_name text)
+CREATE FUNCTION cs_add_column_v1(table_name text, column_name text, cast_as text DEFAULT 'text')
```

Comment:
I think the `DROP` statement also needs to be updated since a new arg was added here.

Reply:
Good catch, thanks